### PR TITLE
Fix go static check problem and optimize code

### PR DESCRIPTION
**Pull request by wongearl**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Fix go static check problem and optimize code
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go [hunk 1]
```diff
@@ -442,10 +442,10 @@ func TestAnnotateErrors(t *testing.T) {
 				return
 			}
 			if bufOut.Len() > 0 {
-				t.Errorf("buffer should be empty: %s", string(bufOut.Bytes()))
+				t.Errorf("buffer should be empty: %s", bufErr.String())
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go:445]: This should be `bufOut`?

Comment (wongearl) [reply staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go]: yes, my fault

Comment (wongearl) [reply staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go]: @ardaguclu i have corrected it，thanks.

**Pull request closed without merging**